packages/flutter_tools/lib/src/localizations/gen_l10n.dart

```diff
@@ -2,6 +2,8 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'dart:io';
```

Comment:
Don't import `dart:io` into the tool. Instead you can use: https://github.com/christopherfujino/flutter/blob/re-enable-packages-autoroller/packages/flutter_tools/lib/src/base/platform.dart#L38